README.md

```diff
@@ -0,0 +1,43 @@
+# My Porfolio
+
+A Porfolio to help showcase my skills and achievments to employers
+
+## Screenshots
+
+<img src="images/mobile-view.png" >
+
+<img src="images/mobile-view2.png">
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+Clone the project
+
+```bash
+  git clone https://github.com/wuyepabdul/microverse-porfolio
+```
+
+Go to the project directory
+
+```bash
+  cd hello-microverse
+```
+
+Run the project
+
+```bash
+  Right click and choose Open file in browser or live server
+
+```
+
+## Authors
+
+- [@Wuyep Abdul](https://www.github.com/wuyepabdul)
+
+## License
+
+This project is [MIT](https://choosealicense.com/licenses/mit/) licensed
```

Comment:
Your project description is good, but there are some sections missing in your repository, you should add more sections to have a professional Github. you can check [This Example](https://github.com/DanteAlonsoHT/OnlineCourse-FInalCapstone) to see how you can type your README file 😉 

- [ ] [OPTIONAL] You always try to make a professional README file.